tests/unit/components/sl-tooltip-test.js

```diff
@@ -189,6 +207,25 @@ test( '"popover" property needs to be a string or undefined', function( assert )
         'Property was empty'
     );
 
+    // Symbol Property
+
+    assertionThrown = false;
+
+    try {
+        let sym = Symbol("foo");
```

Comment:
Modify to follow these style guidelines:
* https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#block-statements
* https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#strings-1